server/lib/feature_flags.py

```diff
@@ -18,8 +18,9 @@
 AUTOCOMPLETE_FEATURE_FLAG = 'autocomplete'
 PLACE_PAGE_EXPERIMENT_FEATURE_FLAG = 'dev_place_experiment'
 PLACE_PAGE_GA_FEATURE_FLAG = 'dev_place_ga'
+BIOMED_NL_ROUTES_FEATURE_FLAG = 'biomed_nl_routes'
```

Comment:
sorry, just catching this now, but why make the flag specifically just for routes? can we do 'biomed_nl', so that it's a flag for the full feature, including the client piece

Reply:
I'm planning on conditionally registering the html and api routes based on this flag. Is that the same a a flag for the full feature?

Reply:
yes, when you register the html and api routes, the full feature should show up. I just meant, we can just name the flag "biomed_nl" instead of "biomed_nl_routes"?